docs/services/nexus/deploy.md

```diff
@@ -0,0 +1,157 @@
+# Deploy to the Nexus
+When joining the CodeMC Project, you will be given a login for our Nexus service. The Nexus Service from CodeMC allows you to distribute your project for others to integrate into their own projects as a library, using their build management tools.
+
+We will explain how you can setup automatic deployments to the Nexus within this page.
+
+/// warning | Important!
+Remember to follow our [Usage Guidelines](../../usage-guidelines.md) on how to use our services to not be punished.
+///
+
+## Prerequisites
+Before you can deploy your project to the Nexus will you first need to do some preparations to ensure that everything will work as expected.
+
+### Login
+You first need to login to your CodeMC Account and head towards the project you own that should have automatic deployments setup.
+
+### Configure Project
+How you configure your project is slightly different depending on the build manager you use.
+
+/// tab | :simple-gradle: Gradle (Freestyle Project)
+1.  Press the :octicons-gear-16: **Configure** button on the left-hand side.
+2.  Scroll to the **Build Environment** Section and press **Use secret text(s) or file(s).
+3.  In the appearing sub-menu, press **Add** and select **Username and password (separated)**
+4.  For the **Username Variable** and **Password Variable** text boxes, set a fitting name to use.
+    //// warning | Important
+    These text boxes are NOT for your actual username and password. They are used to set the variable names that jenkins should use to get the respective values.
+    ////
+5.  Leave the **Credentials** Menu at **jenkins-deploy**
```

Comment:
This is the new name for user-specific credentials:
```suggestion
5.  Leave the **Credentials** Menu at **nexus-repository**
```

Reply:
Is this already implemented? Because last I checked it still was jenkins-deploy.

Reply:
`jenkins-deploy` is the name for the global credentials. Because we're not going to remove it (at least not immediately), I changed the name for the new auto-generated user credentials.